components/dashboard/src/projects/projects.routes.ts

```diff
@@ -4,8 +4,47 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
+import { Project } from "@gitpod/gitpod-protocol";
+import { TabEntry } from "../components/Header";
+
 export const projectsPathMain = "/projects";
 export const projectsPathMainWithParams = [projectsPathMain, ":projectName", ":resourceOrPrebuild?"].join("/");
 
 export const projectsPathInstallGitHubApp = "/install-github-app";
 export const projectsPathNew = "/new";
+
+export function getProjectTabs(project: Project | undefined): TabEntry[] {
+    if (!project) {
+        return [];
+    }
+    const projectSlug = Project.slug(project);
+    return [
+        {
+            title: "Branches",
+            link: `/projects/${projectSlug}`,
+        },
+        {
+            title: "Prebuilds",
+            link: `/projects/${projectSlug}/prebuilds`,
+        },
+        {
+            title: "Settings",
```

Comment:
**thought**: I know this is out of the scope, but if we change the project header we can later on change the org settings to have a fixed header.

Sounds also good as a follow-up issue or PR.  🏓 

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2023-02-02 at 5 28 35 PM (2)" src="https://user-images.githubusercontent.com/120486/216367871-2688d7d2-4ce7-487b-93ec-4b0629dd4679.png">  | <img width="1440" alt="Screenshot 2023-02-02 at 5 29 07 PM (2)" src="https://user-images.githubusercontent.com/120486/216367880-18ff51a9-c859-4b6a-9d12-c1b7d30773e1.png"> |